front/src/TurtleBar/TurtleBar.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import TurtleEnableButton from "./TurtleEnableButton";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+/**
+ * Simple bar with all the turtles
+ * Author: S.A. Tanja
+ * Author: H.E. van der Laan
+ * Author: T.T.P. Franken
+ */
+
+const TurtleBar = props => {
+  return (
+    <div className={props.className}>
+      {props.turtles.map((turtle, position) => {
+        return (
+          <TurtleEnableButton
+            key={turtle.id}
+            enabled={turtle.enabled}
+            id={position + 1}
+            onEnable={() => props.onEnable(position)}
+            onDisable={() => props.onDisable(position)}
+          />
+        );
+      })}
+    </div>
+  );
+};
+
+TurtleBar.propTypes = {
+  turtles: PropTypes.object.isRequired,
+  onDisable: PropTypes.func.isRequired,
+  onEnable: PropTypes.func.isRequired
+};
+
+const TurtleBar = styled(TurtleBar)`
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  overflow-x: auto;
+  align-content: space-between;
+  border-style: solid;
+  border-width: 0rem 0rem 0.2rem 0rem;
+  margin-bottom: 0.2rem;
+  background-color: #ededed;
+  position: fixed;
+  z-index: 9999;
```

Comment:
why 9999?

Reply:
no reason, as long as it's in front. If you have an alternative, tell us, we'll use it.

Reply:
done.